src/components/Migrate/Converter/ConverterFormControls.tsx

```diff
@@ -86,7 +55,7 @@ function BaseConverterFormControls({
   const { updateConvertAmount, conversionType } = useMigrateState()
   const { showAccount } = useAccountModule()
   const { layoutName } = useLayout()
-  const [agree, setAgree] = useState(conversionType !== 'ANJ-LOCK')
```

Comment:
`useState` hook could be deleted entirely, replacing where `agree` with true, where necessary.